src/main/java/com/github/seratch/jslack/api/model/block/SectionBlock.java

```diff
@@ -20,6 +21,7 @@
     private final String type = "section";
     private TextObject text;
     private String blockId;
-    private List<TextObject> fields;
+    @Builder.Default
+    private List<TextObject> fields = new ArrayList<>();
```

Comment:
The reason I didn't initialize the `List<>` fields is because Slack (sometimes) gives errors when it encounters an empty list in the generated JSON. The proper solution if/when you don't want un-initialized fields is to have a Gson type adapter that skips empty lists (e.g. something like https://stackoverflow.com/questions/11942118/how-do-you-get-gson-to-omit-null-or-empty-objects-and-empty-arrays-and-lists)

Reply:
I understand it. I will revert the change with a comment mentioning the reason. Thanks!